E2E-tests/tests/test_rpc.py

```diff
@@ -120,8 +120,10 @@ def test_get_registrations(self, api: BlockchainApi):
         registrations = api.get_registrations(mc_epoch, mainchain_key)
 
         assert isinstance(registrations, list)
-        if registrations:
+        if len(registrations) > 0:
```

Comment:
if registration list is empty test will pass, is this expected?

Reply:
no, really?

len([]) == 0

0 > 0 should be false.

I should check for an empty list?

Reply:
I'll do something like `if registrations is not empty`